Exercise_3.py

```diff
@@ -1,20 +1,41 @@
+# Time Complexity : 
```

Comment:
Please mention the complexities that u feel are correct then i will correct if they are wrong